frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/HealthCard.tsx

```diff
@@ -70,27 +71,24 @@ const HealthCard: React.FC<HealthCardProps> = ({
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>Health</DashboardCardTitle>
+        <DashboardCardTitle>Status</DashboardCardTitle>
       </DashboardCardHeader>
       <DashboardCardBody>
         <HealthBody>
-          <HealthItem state={health.state} message={health.message} />
+          <Gallery className="co-overview-status__health" gutter="md">
```

Comment:
Nit: Gallery and GalleryItem is not needed as we're showing just one HealthItem. But we can keep it to make it consistent with `overview-dashboard/status-card`

Reply:
We'll have two items per the designs.